users/vieting/experiments/switchboard/hybrid/feat/experiments.py

```diff
@@ -213,10 +213,27 @@ def run_specaug_scf():
 
     nn_args = get_nn_args_baseline(
         nn_base_args={
-            "scf": dict(
-                returnn_args=dict(batch_size=7000, specaug_mask_sorting=True),
+            "scf": dict(returnn_args=dict(batch_size=7000, specaug_mask_sorting=True),
```

Comment:
Maybe this already was like this before, but why does this experiment not use gradient accumulation?

Reply:
Fixed